src/dark-mode-toggle.mjs

```diff
@@ -236,8 +236,6 @@ aside {
 </style>
 <form>
 <fieldset>
-  <legend></legend>
-
   <input id="lightRadio" name="mode" type="radio">
   <label id="lightLabel" for="lightRadio"></label>
```

Comment:
We could potentially shorten this quite a bit by wrapping the label around the input, avoiding the need for the `<label for>` attributes (and potentially even the `<input id>`?):

```html
<label id="lightLabel"><input name="mode" type="radio"></label>
```

Or does this complicate the CSS?

Same for the other label+inputs below.

Reply:
Nah, we need this for all of [these rules](https://github.com/GoogleChromeLabs/dark-mode-toggle/blob/be335afdbf37201887ce7c5aec163cb0d9682a94/src/dark-mode-toggle.mjs#L182-L202). This makes the logic really simple and is a common CSS trick.

Reply:
Gotcha.